static/templates/subscription_invites_warning_modal.handlebars

```diff
@@ -0,0 +1,15 @@
+<div id='invites-warning-overlay' class='overlay new-style show' style='z-index:1000;'>
```

Comment:
I was looking at `subscription_stream_privacy_modal` for reference and it doesn't have the `.overlay div`. Am I missing something?